bin/convert_file.py

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/env -S python -B
+
+# Copyright IBM Corp. 2023
```

Comment:
@lipelopesoliveira There is no need to use for newly created files the same copyright header found in older files (pre-opensourcing). This file is not an IBM copyrighted material, because it was just created.